README.md

```diff
@@ -59,8 +60,17 @@ resource "iterative_task" "example" {
   }
   script = <<-END
     #!/bin/bash
-    mkdir results
-    echo "Hello World!" > results/greeting.txt
+
+    # create output directory if needed
+    mkdir -p results
+    # read last result (in case of spot/preemptible instance recovery)
+    if [[ -f results/epoch.txt ]]; then EPOCH="$(cat results/epoch.txt)"; fi
```

Comment:
```suggestion
    if test -f results/epoch.txt; then EPOCH="$(cat results/epoch.txt)"; fi
```

POSIX–friendly?

Reply:
I counter with https://github.com/iterative/terraform-provider-iterative/blob/docs-iter/task/common/machine/script.go#L112

Reply:
1bb47a3 be happy :)

Reply:
🐲🔥✈️

Reply:
If it has to be homogeneous, I don't know if it's better to use `[[` here. I tend to use `test` with `-options` and `[[` with `==` comparison operators. 🙃